app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java

```diff
@@ -146,13 +155,35 @@ private void loadAlbum() {
         PXLAlbumSortOptions so = new PXLAlbumSortOptions();
         so.sortType = PXLAlbumSortType.APPROVED_TIME;
         so.descending = true;
-        album.setPerPage(20);
+        //album.setPerPage(9);
```

Comment:
we dont need per page?

Reply:
The API of Distillery returns 20 items as a default. I just removed it to use the default. But, there is no example of album.setPerPage(..). I'll restore the code.